templates/actors/creature/creature-notes.html

```diff
@@ -1,10 +1,10 @@
-<h2 class="note-section label">Biography</h2>
+<h2 class="note-section label">{{localize "Biography"}}</h2>
 <div class="note-section biography">
   {{editor content=data.details.biography.value target="data.details.biography.value" button=true owner=owner editable=editable}}
 </div>
 
 <div class="inventory-header trait-header">
-  <div id="trait-name">Toggle Creature Trait(s)</div>
+  <div id="trait-name">{{localize "Toggle Creature Trait(s)"}}</div>
```

Comment:
I don't see a key for these (and they should be actual keys, such as `SHEET.ToggleTraits`)